Add `Wiki Contribution Guide` page (#12)
osuTitanic/wiki: wiki/Wiki_Contribution_Guide/en.md

# Titanic! Wiki Contribution Guide

> Thanks for your interest in helping improve the Titanic! wiki. We really appreciate it. 

Reviewer: ```suggestion
Thanks for your interest in helping improve the Titanic! wiki. We really appreciate it.
```

> This is a step by step guide on how you can help make changes to the wiki.

Reviewer: ```suggestion
This is a step-by-step guide on how you can help make changes to the wiki.
```

> To begin with you will need to have a basic understanding of GitHub or [git](https://git-scm.com/). It's okay if you get confused you can always ask in the [Titanic! discord](https://discord.gg/qupv72e7YH) in the `#wiki` channel.

Reviewer: ```suggestion
To begin with you will need to have a basic knowledge of GitHub or [git](https://git-scm.com/). It's okay if you get confused, you can always ask in the [Titanic! Discord server](https://discord.gg/qupv72e7YH) in the `#wiki` channel.
```
